diff --git a/build/android/gyp/process_resources.py b/build/android/gyp/process_resources.py
index d227954..7c489dc 100755
--- a/build/android/gyp/process_resources.py
+++ b/build/android/gyp/process_resources.py
@@ -363,7 +363,8 @@ def main():
       package_command += ['-G', options.proguard_file]
     if options.shared_resources:
       package_command.append('--shared-lib')
-    build_utils.CheckOutput(package_command, print_stderr=False)
+    print(package_command)
+    build_utils.CheckOutput(package_command, print_stderr=True, print_stdout=True)

     if options.extra_res_packages:
       CreateExtraRJavaFiles(
